mail_notification_with_history/models/mail_message.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2022 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+
+from odoo import models
+
+
+class Message(models.Model):
+    _inherit = "mail.message"
+
+    def get_notification_message_history(self):
```

Comment:
```suggestion
    def _get_notification_message_history(self):
```
See comment below about public methods

Reply:
changed.